packages/eas-cli/scripts/prepack.js

```diff
@@ -0,0 +1,14 @@
+const { exec } = require('child_process');
```

Comment:
use @expo/spawn-async, it's already in deps

Reply:
I have initially went with that package, but it produces an ESLint error which need to be suppressed, so I just used Node API instead:
<img width="585" alt="Untitled" src="https://user-images.githubusercontent.com/719641/184363330-beebe3b9-8a66-49b8-830c-26ce34c26fb9.png">